### DISCUSSION TITLE
sub process and file access ?

### DISCUSSION THREAD

**Discussion by quertenmont** [gramineproject/gramine#1081] [Q&A]
Hello,

I am facing a new issue and would like to know what's the best way to proceed :-)
Thanks in advance for your help

So I have a GSC graminified container running a python script basically.
My container has some encrypted files defined with:
```
fs.mounts = [
  { type = "encrypted", path = "/resources", uri = "file:/resources", key_name = "_sgx_mrsigner" },
]
```
My python script has to spawn a subprocess (to run a shell command).
I see that the subcommand is properly being run in an enclave, but it seems that it can't access the encrypted directory "/resources"

Here an example with an "ls" command.  I know I can list the files from python directly ( and this works without issues).
but the ls command here is just the simplest example I can come with to reproduce the issue (I'll have to run more sophisticated method than ls).

run_cmd: 'ls resources/'
Detected a huge manifest, preallocating 64MB of internal memory.
Detected a huge manifest, preallocating 64MB of internal memory.
ls: cannot access 'resources/': No such file or directory

Is there a way to tell the enclaved subprocess to mount the encrypted files?  (remember I am using GSC and gramine 1.3.1).

Thanks for helping,
Loic

**Comment by mkow** [comment]
1. Please report issues only for the current master, not any older versions.
2. Does this directory work in the parent process?

**Comment by quertenmont** [comment]
Hi @mkow 

I don't get your point 1., isn't 1.3.1 not the latest version?

regarding 2., yes it works in the parent process.

**Reply by mkow** [reply]
The latest version is `master` branch in the repo, which already have 72 more commits than 1.3.1. Also, please use threading when replying (you started a new thread instead of replying to mine).

> regarding 2., yes it works in the parent process.

Could you increase the log level in the manifest and check the logs again? Also, how do you test it in the parent?

**Comment by boryspoplawski** [comment]
Try doing `ls /resources` (add leading `/`)
